Sources/TuistKit/Lint/CodeLinter.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+protocol CodeLinting {
+    func lint(sources: AbsolutePath, path: AbsolutePath) throws
+}
+
+class CodeLinter: CodeLinting {
+    private let rootDirectoryLocator: RootDirectoryLocating
+    private let binaryLocator: BinaryLocating
+
+    init(rootDirectoryLocator: RootDirectoryLocating = RootDirectoryLocator(),
+         binaryLocator: BinaryLocating = BinaryLocator())
+    {
+        self.rootDirectoryLocator = rootDirectoryLocator
+        self.binaryLocator = binaryLocator
+    }
+
+    // MARK: - CodeLinting
+
+    func lint(sources: AbsolutePath, path: AbsolutePath) throws {
+        let swiftLintPath = try binaryLocator.swiftLintPath()
+        let swiftLintConfigPath = self.swiftLintConfigPath(path: path)
+        let swiftLintArguments = buildSwiftLintArguments(swiftLintPath: swiftLintPath,
+                                                         sources: sources,
+                                                         configPath: swiftLintConfigPath)
+
+        let result = try System.shared.observable(swiftLintArguments)
+            .mapToString()
+            .toBlocking()
+            .toArray()
```

Comment:
If you follow my suggestion below, I think you can do `.last()` instead of `.toArray()` and ignore the returned object.